Signal/src/ViewControllers/ConversationView/Cells/OWSContactOffersCell.m

```diff
@@ -62,19 +66,48 @@ - (void)commontInit
         [self createButtonWithTitle:NSLocalizedString(@"CONVERSATION_VIEW_UNKNOWN_CONTACT_BLOCK_OFFER",
                                         @"Message shown in conversation view that offers to block an unknown user.")
                            selector:@selector(block)];
+
+    UIStackView *buttonStackView = [[UIStackView alloc] initWithArrangedSubviews:@[
+        self.addToContactsButton,
+        self.addToProfileWhitelistButton,
+        self.blockButton,
+    ]];
+    buttonStackView.axis = UILayoutConstraintAxisVertical;
+    buttonStackView.spacing = self.vSpacing;
+    // Ensure all of the buttons have the same width.
+    buttonStackView.alignment = UIStackViewAlignmentFill;
+    buttonStackView.layoutMargins = UIEdgeInsetsZero;
```

Comment:
For stack views, unless you're setting `isLayoutMarginsRelativeArrangement` I don't think layoutMargins matter.

Reply:
Removed.